themes/custom/campaign_base/sass/styles.scss

```diff
@@ -7,3 +7,5 @@ $image-path: "/profiles/contrib/cr/themes/custom/campaign_base/images/patternlab
 @import "@comicrelief/pattern-lab/sass/base/_variables";
 @import "@comicrelief/pattern-lab/sass/base/_core";
 @import "@comicrelief/pattern-lab/sass/base/_components";
+
+//test change do not merge me :) 
```

Comment:
@Heleen-cr @AndyEPhipps note that you can also use the develop environment on platform.sh for this, which should contain the latest `develop`. This is equal to what you're doing here but without adding a new env